Formula/jupp.rb

```diff
@@ -0,0 +1,35 @@
+class Jupp < Formula
+  desc "Professional screen editor for programmers"
+  homepage "https://www.mirbsd.org/jupp.htm"
+  url "https://www.mirbsd.org/MirOS/dist/jupp/joe-3.1jupp40.tgz"
+  version "3.1jupp40"
+  sha256 "4bed439cde7f2be294e96e49ef3e913ea90fbe5e914db888403e3a27e8035b1a"
+  license "GPL-1.0-or-later"
+  # Upstream HEAD in CVS: http://www.mirbsd.org/cvs.cgi/contrib/code/jupp/
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "expect" => :test
+  uses_from_macos "ncurses"
+  on_macos do
+    depends_on "gnu-sed" => :build
+  end
+
+  def install
+    if OS.mac?
+      inreplace "Makefile.am", "sed ", "#{Formula["gnu-sed"].opt_bin}/gsed"
+      inreplace "Makefile.in", "sed ", "#{Formula["gnu-sed"].opt_bin}/gsed"
+    end
+    system "autoreconf", "-vfi"
+    system "sh", "./configure", "--prefix=#{prefix}",
+                                "--enable-sysconfjoesubdir=/jupp",
+                                "--disable-dependency-tracking"
+    system "make", "install"
+  end
+
+  test do
+    assert_match "File (Unnamed) not changed so no update needed.",
+      shell_output("echo 'spawn #{bin}/jupp;send \"q\";expect eof'|env TERM=tty expect -")
```

Comment:
```suggestion
      pipe_output("env TERM=tty expect -", "spawn #{bin}/jupp;send \"q\";expect eof'")
```

Reply:
Fixed in https://github.com/Homebrew/homebrew-core/pull/106097/commits/4c52a91b120a06e0982af237f7a649175fbc75a6.